app/mailers/emailer.rb

```diff
@@ -31,13 +38,14 @@ def new_feature_adoption_to_admin
 
   def user_email_verification(user)
     @email_verification_url = email_verification_url(user.email_verification_token)
-    @user_contact_name = user.contact_name
+    @user_contact_name = user.contact_name.presence || user.email
     @user_email = user.email
     generic_user_html_email(user, __method__)
   end
 
   def user_approval_removed(user)
     @user = user
+    @user_contact_name = user.contact_name.presence || user.email
```

Comment:
These could be DRY'd up by adding a `contact_name(user)` method to this class (or even to the `User` class, if we would need it elsewhere). That said, these will no longer be needed once we require contact names, so it's up to you.

Reply:
I agree and would have done that. But just as we had discussed, right going to just follow the current pattern so we can get this fix. Cleanup will have to happen later.